tests/console/commands/test_add.py

```diff
@@ -32,16 +32,25 @@
 
 @pytest.fixture
 def poetry_with_up_to_date_lockfile(
-    project_factory: ProjectFactory, fixture_dir: FixtureDirGetter
+    project_factory: ProjectFactory,
+    fixture_dir: FixtureDirGetter,
+    repo: TestRepository,
+    command_tester_factory: CommandTesterFactory,
 ) -> Poetry:
     source = fixture_dir("up_to_date_lock")
 
-    return project_factory(
+    poetry = project_factory(
         name="foobar",
         pyproject_content=(source / "pyproject.toml").read_text(encoding="utf-8"),
         poetry_lock_content=(source / "poetry.lock").read_text(encoding="utf-8"),
     )
 
+    # ensure poetry.lock with updated syntax
+    repo.add_package(get_package("docker", "4.3.1"))
+    command_tester_factory("lock", poetry=poetry).execute()
```

Comment:
This change was required because the existing lock file is using an older version of the lock file.

I need to have the same version in order to compare that the lock file was untouched.